### PR TITLE
fix description

### PR DESCRIPTION
**Pull request by loverto**

### CODE REVIEW

File: content/docs/reference/config/installation-options/index.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ To customize Istio install using Helm, use the `--set <key>=<value>` option in H
 | `gateways.istio-ingressgateway.resources` | `{}` |  |
 | `gateways.istio-ingressgateway.loadBalancerIP` | `""` |  |
 | `gateways.istio-ingressgateway.serviceAnnotations` | `{}` |  |
-| `gateways.istio-ingressgateway.type` | `LoadBalancer #change to NodePort, ClusterIP or LoadBalancer if need be` |  |
+| `gateways.istio-ingressgateway.type` | `LoadBalancer` |  `#change to NodePort, ClusterIP or LoadBalancer if need be` |
```
Comment (frankbu) [inline content/docs/reference/config/installation-options/index.md]: should also remove `#` at start of description

File: content/docs/reference/config/installation-options/index.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ To customize Istio install using Helm, use the `--set <key>=<value>` option in H
 | `gateways.istio-egressgateway.autoscaleMin` | `1` |  |
 | `gateways.istio-egressgateway.autoscaleMax` | `5` |  |
 | `gateways.istio-egressgateway.serviceAnnotations` | `{}` |  |
-| `gateways.istio-egressgateway.type` | `ClusterIP #change to NodePort or LoadBalancer if need be` |  |
+| `gateways.istio-egressgateway.type` | `ClusterIP` | `#change to NodePort or LoadBalancer if need be` |
```
Comment (frankbu) [inline content/docs/reference/config/installation-options/index.md]: see above comment

File: content/docs/reference/config/installation-options/index.md [hunk 3]
```diff
@@ -160,7 +160,7 @@ To customize Istio install using Helm, use the `--set <key>=<value>` option in H
 | `ingress.autoscaleMax` | `5` |  |
 | `ingress.service.annotations` | `{}` |  |
 | `ingress.service.loadBalancerIP` | `""` |  |
-| `ingress.service.type` | `LoadBalancer #change to NodePort, ClusterIP or LoadBalancer if need be` |  |
+| `ingress.service.type` | `LoadBalancer` |  `#change to NodePort, ClusterIP or LoadBalancer if need be` |
```
Comment (frankbu) [inline content/docs/reference/config/installation-options/index.md]: see above comment

File: content/docs/reference/config/installation-options/index.md [hunk 4]
```diff
@@ -230,7 +230,7 @@ To customize Istio install using Helm, use the `--set <key>=<value>` option in H
 | --- | --- | --- |
 | `security.replicaCount` | `1` |  |
 | `security.image` | `citadel` |  |
-| `security.selfSigned` | `true # indicate if self-signed CA is used.` |  |
+| `security.selfSigned` | `true` |  `# indicate if self-signed CA is used.` |
```
Comment (frankbu) [inline content/docs/reference/config/installation-options/index.md]: see above comment

**Comment by geeknoid** [comment]
This is a generated file, any changes in here will be lost the next time the file is regenerated. See the comment at the top of the file for details.

**Comment by frankbu** [comment]
It looks like this would need to be fixed in the generator https://github.com/istio/istio.github.io/blob/master/scripts/tablegen.py

@sdake WDYT?

**Comment by sdake** [comment]
@loverto thanks for the submission.  The root problem is that the `tablegen.py` script is deficient.  This code does not currently process comments from `values.yaml` and add them to the description field.  It would be nice if it did as that was the original intent.  @gyliu513 and @sdake are original authors of this script.  We just haven't had time to finish the job on it.  If you want to take a look at it, would be happy to review/test/etc.

Cheers
-steve

**Comment by sdake** [comment]
@frankbu ya - the table generator is incomplete.  I've filed an issue here: https://github.com/istio/istio.github.io/issues/2566 to better enable the correction of this script.

Cheers
-steve

**Comment by sdake** [comment]
@loverto I'm going to close this PR since this is autogenerated output and any changes or improvements made to this file will be overwritten.

Thanks for the contribution and keep at it :)

**Pull request closed without merging**